docs/src/remap/remap.adoc

```diff
@@ -766,14 +766,14 @@ We'll use the following correspondence in our example:
 [frame="topbot",grid="none"]
 [options="header"]
 |===
-|iocontrol.0 pin  |motion pin
-|tool-prepare     |digital-out-00
-|tool-prepared    |digital-in-00
-|tool-change      |digital-out-01
-|tool-changed     |digital-in-01
-|tool-prep-number |analog-out-00
-|tool-prep-pocket |analog-out-01
-|tool-number      |analog-out-02
+|`iocontrol.0` pin    |`motion` pin
```

Comment:
why not a leading "m" in this line?

Reply:
because of the word "pin" that is not part of the pin name

Reply:
ok